db/seeds.rb

```diff
@@ -5,3 +5,6 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
```

Comment:
Feel free to delete auto-generated comments like this. They tend to clutter up the code.